src/BenchmarksApps/Mvc/ProductsController.cs

```diff
@@ -0,0 +1,126 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Mvc
+{
+    public class ProductsController : ControllerBase
```

Comment:
API controller?

Reply:
I don't want to add those behaviors just yet.

We can add more variations later.